### DISCUSSION TITLE
希望能支持P站官方的TAG

### DISCUSSION THREAD

**Discussion by SnowMoonSS** [bot-ssttkkl/nonebot-plugin-pixivbot#166] [Ideas]
![image](https://user-images.githubusercontent.com/21017431/234814219-8d464c02-769b-4718-a9f4-3a0168ca50ac.png)
不知称呼这个为官方tag是否合适。
AI生成的作品越来越多，抽中AI作品的概率也越来越高，作者为自己打的AI TAG也千奇百怪，不知这个“官方TAG”一样的东西能否利用一下，这样在屏蔽列表里只加这一个“AI生成”就能解决大部分AI作品……
```
pixiv_block_tags=["NovelAI","AI生成","StableDiffusion","AIart","AIイラスト","AI绘图"]
```

**Comment by ssttkkl** [comment]
这个所谓的“官方TAG”其实只是靠写死的几个字段标识的，只需要过滤AI图的话是可以实现的

**Comment by ssttkkl** [comment]
1.8.3版本支持过滤AI图，设置配置项`pixiv_exclude_ai_illusts=true`启用

**Reply by SnowMoonSS** [reply]
我超，我看了两遍都没看到，星际了。谢谢

**Reply by ssttkkl** [reply]
因为是我刚刚更新的（）
